addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -284,7 +284,7 @@ var KanbanRenderer = BasicRenderer.extend({
      * @param {DocumentFragment} fragment
      */
     _renderExampleBackground: function (fragment) {
-        var $background = $(qweb.render('KanbanView.ExamplesBackground'));
+        var $background = $(qweb.render('KanbanView.ExamplesBackground', {model: this.state.model}));
```

Comment:
Passing the model is not very generic and will not help other modules defining custom column names of their own.

Maybe pass to the template a "ghostColumns" parameter that is set to `[_.str.sprintf(_t("Column %s"), 1), _.str.sprintf(_t("Column %s"), 2), ...]` in the base web code and override the '_renderExampleBackground' method in the project module to check that the model is "project.task" and change that ghostColumns parameter to [_t('New'), _t('Assigned'), ...]

That way you don't even have to override the template in the project module, only the JS method.

Does that make sense? Don't hesitate to ask if you have questions.

Reply:
Implement with a `_.range(1, 5)` too to avoid copying `_.str.sprintf(_t("Column %s"), X)` too much.